### ISSUE TITLE
[Question] Transfer bad pokemons

### ISSUE THREAD

**Issue by Dreker** [finndev/PokeBuddy#378]
Hi,

What is considered a bad pokemon?

Thanks for the bot :D

**Comment by Rab89** [comment]
personally i dont use it since some people have reported losing over 1000 CP pokemon actually being transferred.

use at own risk though

**Comment by jersonortizx** [comment]
THE BOT IS WORKING NOW?

**Comment by Dreker** [comment]
@Rab89  hmmm ok, thanks!

@jersonortizx  yes.

**Comment by jersonortizx** [comment]
thankss ::d

**Comment by oOBlueArrowOo** [comment]
it transfers the lowest one of that pokemon

**Comment by keytonic** [comment]
i've noticed that pokebuddy will transfer poke's regardless of if they are in the don not transfer list so i dont use this feature either.

**Comment by Dreker** [comment]
@oOBlueArrowOo 
Example: if i have only 1 pickachu cp 10, will never be transfered?

**Comment by TheWonderSpud** [comment]
@Dreker No, but if you had a pikachu cp 10 and a pikachu cp 11 for example, the pikachu with cp 10 would be transfered.

**Comment by jersonortizx** [comment]
best latitude or longitude? the bot dont find pokeparades :C

**Comment by WolfyxBerg** [comment]
Santa Monica Pier Latitude:	34.0093515
Santa Monica Pier Longitude:	-118.49746820000001
